fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadTask.java

```diff
@@ -73,13 +73,8 @@ protected void exec() {
             // callback on pending task finished
             callback.onTaskFinished(attachment);
             isFinished = true;
-        } catch (LoadException e) {
-            failMsg.setMsg(e.getMessage() == null ? "" : e.getMessage());
-            LOG.warn(new LogBuilder(LogKey.LOAD_JOB, callback.getCallbackId())
-                    .add("error_msg", "Failed to execute load task").build(), e);
         } catch (StarRocksException e) {
             failMsg.setMsg(e.getMessage() == null ? "" : e.getMessage());
-            failMsg.setCancelType(FailMsg.CancelType.USER_CANCEL);
             LOG.warn(new LogBuilder(LogKey.LOAD_JOB, callback.getCallbackId())
                     .add("error_msg", "Failed to execute load task").build(), e);
         } catch (Exception e) {
```

Comment:
what's the difference between `StarRocksException` and `Exception` after remove `failMsg.setCancelType`?

Reply:
https://github.com/StarRocks/starrocks/pull/51072
this pr adds `LoadException`(timeout related) and set `USER_CANCEL` when catch `StarRocksException`, but this is not really user cancel, just not retry when meet resource group pending timeout.

`Exception` is unknown Exception, such as exception caused by bug.